astropy/cosmology/funcs/comparison.py

```diff
@@ -29,7 +29,7 @@
 _FormatsT = Union[_FormatType, Tuple[_FormatType, ...]]
```

Comment:
Weird, why does it not use lower-case `tuple` here?

Reply:
I think it's because it the analyzer cannot determine whether this is a type alias or runtime variable.